starport/cmd/generate.go

```diff
@@ -0,0 +1,19 @@
+package starportcmd
+
+import "github.com/spf13/cobra"
+
+// NewGenerate returns a command that groups code generation related sub commands.
+func NewGenerate() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "generate [command]",
+		Short:   "Generate code",
```

Comment:
People may wonder what is the difference with the scaffold namespace

Could we consider it as generating code outside the core Cosmos SDK modules?

Reply:
Yes, the different in between is a bit ambigous. `scaffold` is more related to adding to/changing in your applications source code whereas `generate` is more like a tool to generate various things spesific to your chain.

Reply:
`generate` commands are idempotent yo

Reply:
Yeah, as Ilker said, you generate code when it's not meant to be edited. Similarly to how proto files generate `.pb.go` and such. I agree that the difference is subtle, but having both in the same namespace doesn't sound good either.

Reply:
I didn't say we should have the same namespace. Also, `generate` makes totally sense, my comment was just about making the `Short` description more explicit